data/com/mojang/math/Matrix3f.mapping

```diff
@@ -3,5 +3,39 @@ CLASS com/mojang/math/Matrix3f
 		COMMENT Copy constructor for this class.
 		ARG 1 other
 			COMMENT The instance to copy from
+	METHOD <init> (Lcom/mojang/math/Matrix4f;)V
+		ARG 1 other
```

Comment:
Not really the `other` matrix. It drops the 4th dimension of the 3x3, like becoming the normal or affine transform and throwing out the projected data in some higher plane dimension.

Reply:
I feel like I got bitten by this same mistake in the past, because of the similarity of `Matrix3f` and `Matrix4f` in naming.